universal-application-tool-0.0.1/app/views/questiontypes/FileUploadQuestionRenderer.java

```diff
@@ -68,12 +76,15 @@ private ContainerTag signedFileUploadFields(ApplicantQuestionRendererParams para
         .with(input().withType("hidden").withName("X-Amz-Date").withValue(request.date()))
         .with(input().withType("hidden").withName("Policy").withValue(request.policy()))
         .with(input().withType("hidden").withName("X-Amz-Signature").withValue(request.signature()))
-        .with(input().withType("file").withName("file").attr(Attr.ACCEPT, "image/*,.pdf"))
+        .with(input().withType("file").withName("file").attr(Attr.ACCEPT, acceptFileType()))
         .with(errorDiv(params.messages()));
   }
 
+  private String acceptFileType() {
```

Comment:
(nit) this could be a string constant instread.

Reply:
Done.